.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: "module",
+        ecmaFeatures: {
+        }
+    },
+    rules: {
+        "max-len": ["warn", {
+            code: 90,
+            ignoreComments: true,
+        }],
+        curly: ["error", "multi-line"],
+    }
```

Comment:
Interestingly, turning these on produces 537 warnings, so presumably it's somewhat stricter than jshint :/

Should we turn them on and aspire to fix the warnings?